src/helix/client/client_ext.rs

```diff
@@ -937,6 +937,78 @@ impl<'client, C: crate::HttpClient + Sync + 'client> HelixClient<'client, C> {
         }
     }
 
+    /// Get the emotes of the user.
```

Comment:
```suggestion
    /// Get all emotes accessible to the user in all chats.
```

Could be interpreted as "emotes created by the user".